core/common/src/main/java/alluxio/metrics/MetricKey.java

```diff
@@ -2426,6 +2426,13 @@ public static String getSyncMetricName(long mountId) {
           .setIsClusterAggregated(false)
           .build();
 
+  public static final MetricKey CLIENT_UFS_FALLBACK_COUNT =
+      new Builder("Client.UfsFallBackCount")
+          .setDescription("The number of fallbacks to UFS in Dora when opening files.")
```

Comment:
Dora is an internal name not external name.
The number of fallbacks to UFS when failing to open file in Alluxio distributed cache

Reply:
Gotcha! I updated the description based on your comment. Feel free to let me know whether it looks good to you.